util/entropy.c

```diff
@@ -55,7 +55,7 @@ entropy_read(uint8_t * buf, size_t buflen)
 
 		/* We've filled a portion of the buffer. */
 		buf += lenread;
```

Comment:
I'd cast lenread to (size_t) both times, even if clang doesn't care about pointer arithmetic.